draft-ietf-rats-eat.md

```diff
@@ -1136,12 +1136,17 @@ The following sub-sections define the three types for representing submodules:
 
 When decoding a submodule in a CBOR-encoded EAT, the Claims-Set type will be encoded as a map, the Detached-Submodule-Digest type as an array, and the Nested-Token type as a CBOR-tagged object.
```

Comment:
Nested-Token is a tstr or a bstr in CBOR.

Reply:
As was covered shortly below these words. As noted above, now these words and the former Nested-Token section are together.